### ISSUE TITLE
Improve contrast in form fields in Django Admin

### ISSUE THREAD

**Issue by kimberly-a** [cisagov/manage.get.gov#3126]
### Story

As a user with low vision, color blindness, or other visual impairments, I need form field boxes to have sufficient contrast with the surrounding background so I can easily identify where to enter information and avoid confusion or errors.

### Acceptance Criteria

- [ ] Change form field borders in Django admin from `#f0f0f0` to Base Dark `#565C65` in light mode.
- [ ] Change form field borders from `#353535` to Base Light `#A9AEB1` in dark mode.

![Image](https://github.com/user-attachments/assets/9b12d349-c9f2-4d5e-8dad-b8c6842b2d65)

### Additional Context

- Users with low vision, color blindness, or other visual impairments may struggle to see form fields with insufficient contrast, leading to frustration and difficulty navigating content.
- Form field borders must meet a contrast ratio of at least 3:1 against the adjacent background based on WCAG AA guidelines.
- In light mode, our current form field's contrast ratio is 1.13:1, which fails WCAG AA guidelines. Changing our form field borders from `#f0f0f0` to Base Darker `#565C65` in light mode results in a 6.74:1 contrast ratio, which passes WCAG AA guidelines.
-  In dark mode, our current form field's contrast ratio is 1.52:1, which fails WCAG AA guidelines. Changing our dark mode form field borders from `#353535` to Base Light `#A9AEB1` results in a 8.36:1 contrast ratio, which passes WCAG AA guidelines.

### Issue Links

- Relates to #1522

**Comment by allly-b** [comment]
It looks like design did work looking at contrast in django admin, but only for form fields, where as we have  [2209](https://github.com/cisagov/manage.get.gov/issues/2209) specifically to fix dark mode issues across the board all at once to avoid the issues we have had in the past. In the past, we tried changing only a couple contrast/coloration issues in django, like this ticket suggests doing, but that typically results in the changes being lost in follow on contrast/coloration issues or in other side effects occurring. Given we know we have several issues related to dark mode especially they should be grouped together.
